--- libvisual-0.4.0/libvisual/Makefile.am.orig	2006-02-20 09:20:58.000000000 +0100
+++ libvisual-0.4.0/libvisual/Makefile.am	2006-03-22 19:25:47.953499500 +0100
@@ -19,8 +19,6 @@
 
 DEFS = @DEFS@
 
-LIBS += -L. -L$(libdir) $(LV_THREAD_LIBS) @LTLIBINTL@
-
 PLUGPATH = $(libdir)/libvisual
 
 AM_CFLAGS = -DPLUGPATH="\"$(LIBVISUAL_PLUGINS_BASE_DIR)\"" \
@@ -30,6 +28,7 @@
 	   -I$(top_builddir) -I$(top_builddir)/libvisual
 
 libvisual_@LIBVISUAL_VERSION_SUFFIX@_la_LDFLAGS = -export-dynamic -no-undefined
+libvisual_@LIBVISUAL_VERSION_SUFFIX@_la_LIBADD = $(LV_THREAD_LIBS) @LTLIBINTL@
 
 libvisual_@LIBVISUAL_VERSION_SUFFIX@_la_SOURCES = lv_actor.c lv_input.c lv_event.c \
 		       lv_bin.c lv_plugin.c lv_video.c lv_video_simd.c lv_mem.c \
